### DISCUSSION TITLE
posts/cherry

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#388] [Announcements]
# posts/cherry

20220922 #主义主义支线 #辩证法 Source: 【一小时哲学】“不”字的四种写法，你知道几种？(何为否定) (https://www.bilibili.com/video/BV1Vz4y1r7b2) 名词解释 1. 否定 否定是一种非的态度。“否定性”“无”，即实在性的对立、不同一的规定。 2. 不可能性 不可能性指在两个对立命题中不属于任...

https://notes.alderwish.tk/posts/cherry.html

**Comment by Mapleshade20** [comment]
test
